INFO     root:api.py:98 Invalid event_history_id eeb844b3-4333-4377-bc75-dfd5c60e8a35 provided